src/inmanta/ast/statements/define.py

```diff
@@ -347,7 +347,7 @@ class DefineRelation(DefinitionStatement):
 
     def __init__(self, left, right, annotations=[]):
         DefinitionStatement.__init__(self)
-        self.annotations = annotations
+        self.annotations = [str(x) for x in annotations]
```

Comment:
Why perform str(x)? If annotations should strings instead of references, it would be better to change this in the parser

Reply:
no, it will need to be a resolved reference. This is a temporary workaround

Reply:
ok, I can code up te resolution is required